mlflow/utils/string_utils.py

```diff
@@ -5,15 +5,11 @@
 
 
 def strip_prefix(original, prefix):
-    if original.startswith(prefix):
-        return original[len(prefix) :]
-    return original
+    return original.lstrip(prefix)
```

Comment:
This change alters the original behavior.

```python
>>> "aabbcc".lstrip("abc")
''
```

Reply:
https://docs.python.org/3/library/stdtypes.html?highlight=strip#str.removeprefix was added in python 3.9, but we can't use it because the minimum python version mlflow supports is 3.8.